test/index.test.js

```diff
@@ -256,8 +260,8 @@ describe('Index Tests', () => {
         ...input,
       };
       await assert.rejects(
-        async () => main(params),
-        /TypeError: Cannot read property 'result' of undefined/,
+        () => main(params),
+        { name: 'TypeError' },
```

Comment:
The message `Cannot read property ...` changed in node 12, so just check the error type.